src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -719,10 +719,14 @@ protected function renderHttpException(HttpExceptionInterface $e)
         $this->registerErrorViewPaths();
 
         if ($view = $this->getHttpExceptionView($e)) {
-            return response()->view($view, [
-                'errors' => new ViewErrorBag,
-                'exception' => $e,
-            ], $e->getStatusCode(), $e->getHeaders());
+            try {
+                return response()->view($view, [
+                    'errors' => new ViewErrorBag,
+                    'exception' => $e,
+                ], $e->getStatusCode(), $e->getHeaders());
+            } catch (Throwable $t) {
+                config('app.debug') || throw $t;
```

Comment:
It would make sense to me to still `report()` the exception so that it's not hidden. This should have the effect of writing it to the log or an error tracker (once) so it's still noticeable but doesn't end up in a loop (this has bitten me too before).

```suggestion
                if (config('app.debug')) {
                    throw $t;
                }
                
                $this->report($t);
```